.github/workflows/scan-images.yaml

```diff
@@ -0,0 +1,87 @@
+name: Scan Images for Vulnerabilities (Trivy)
+on:
+  schedule:
+    - cron: "0 9 * * *" # Every day @ 09:00 UTC, run Trivy scans against main
+  workflow_dispatch:
+    inputs:
+      scan_main:
+        description: "Run Trivy scans against images built on `main`."
+        type: boolean
+        required: true
+        default: true
+      version:
+        description: "Released Eraser version to run Trivy scans against. Leave blank for no-op."
+        type: string
+        required: false
+        default: ""
+
+permissions: read-all
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  setup_trivy:
+    name: "Setup"
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    permissions:
+      contents: read
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@8ca2b8b2ece13480cda6dacd3511b49857a23c09
+        with:
+          egress-policy: audit
+      - name: Download trivy
+        run: |
+          pushd $(mktemp -d)
+          wget https://github.com/aquasecurity/trivy/releases/download/v${{ env.TRIVY_VERSION }}/trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
+          tar zxvf trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
+          echo "$(pwd)" >> $GITHUB_PATH
+        env:
+          TRIVY_VERSION: "0.28.0"
```

Comment:
```suggestion
          TRIVY_VERSION: "0.45.0"
```

Reply:
Switched to using the [trivy github action](https://github.com/aquasecurity/trivy-action) for our scans. Pinned the latest released version which [uses trivy 0.45.0](https://github.com/aquasecurity/trivy-action/commit/fbd16365eb88e12433951383f5e99bd901fc618f).